ctl/string.cc

```diff
@@ -91,16 +91,13 @@ string::reserve(size_t c2) noexcept
     if (!isbig()) {
         if (!(p2 = (char*)malloc(c2)))
             __builtin_trap();
-        memcpy(p2, data(), size());
-        p2[size()] = 0;
+        memcpy(p2, data(), size() + 1);
```

Comment:
wouldn't it be faster to just memcpy by a constant the whole thing?

Reply:
oh yeah good point

Reply:
shaves 1ns off the append benchmark that does the initial jump to big string so yeah good catch.

(some others slightly increased but that's gotta be just noise)